torch/_meta_registrations.py

```diff
@@ -1139,6 +1139,33 @@ def meta_zero_(self):
     return self
 
 
+@register_meta(
+    [
+        aten.mul_.Scalar,
+        aten.div_.Scalar,
+        aten.mul_.Tensor,
+        aten.div_.Tensor,
+        aten.logical_and_.default,
+        aten.logical_or_.default,
+        aten.logical_xor_.default,
+    ],
+)
+def meta_binop_inplace(self, other):
```

Comment:
prims impl should have taken care of these...
Or is inplace version missing?

Reply:
Inplace versions are missing!